compiler/onecc-docker/tests/onecc-docker_001.test

```diff
@@ -28,7 +28,7 @@ trap_err_onexit()
 trap trap_err_onexit ERR
 
 configfile="onecc-docker_001.cfg"
-outputfile="inception_v3.opt.circle"
+outputfile="mobilenet_v1_0.5_224.circle"
```

Comment:
please refer https://github.com/Samsung/ONE/blob/master/compiler/one-cmds/tests/prepare_test_materials.sh for really tiny model.